code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm

```diff
@@ -0,0 +1,354 @@
+		return
+
+/obj/structure/elite_tumor/proc/arena_checks()
+	if(src) //Checking to see if we still exist
+		INVOKE_ASYNC(src, .proc/arena_trap)  //Gets another arena trap queued up for when this one runs out.
+		INVOKE_ASYNC(src, .proc/border_check)  //Checks to see if our fighters got out of the arena somehow.
+		INVOKE_ASYNC(src, .proc/fighters_check)  //Checks to see if our fighters died.
+	return
+		
+/obj/effect/temp_visual/elite_tumor_wall
+	name = "magic wall"
+	icon = 'icons/turf/walls/hierophant_wall_temp.dmi'
+	icon_state = "wall"
+
+	duration = 50
+	smooth = SMOOTH_TRUE
+	layer = BELOW_MOB_LAYER
+	var/mob/living/carbon/human/activator = null
+	var/mob/living/simple_animal/hostile/asteroid/elite/ourelite = null
+	color = rgb(255,0,0)
+	light_range = MINIMUM_USEFUL_LIGHT_RANGE
+	light_color = LIGHT_COLOR_RED
+	
+/obj/effect/temp_visual/elite_tumor_wall/Initialize(mapload, new_caster)
+	. = ..()
+	queue_smooth_neighbors(src)
+	queue_smooth(src)
+
+/obj/effect/temp_visual/elite_tumor_wall/Destroy()
+	queue_smooth_neighbors(src)
+	return ..()
+
+/obj/effect/temp_visual/elite_tumor_wall/CanPass(atom/movable/mover, turf/target)
+	if(mover == ourelite || mover == activator)
+		return FALSE
+	else
+		return TRUE
+		
+/obj/structure/elite_tumor/proc/border_check()
+	if(activity == TUMOR_ACTIVE)
+		if(activator != null && !QDELETED(activator) && get_dist(src, activator) >= 12)
+			if(loc)
+				activator.forceMove(loc)
+				visible_message("<span class='boldwarning'>[activator] suddenly reappears above the tumor!</span>")
+				playsound(loc,'sound/effects/phasein.ogg', 200, 0, 50, TRUE, TRUE)
+		if(mychild != null && !QDELETED(mychild) && get_dist(src, mychild) >= 12)
+			if(loc)
+				mychild.forceMove(loc)
+				visible_message("<span class='boldwarning'>[mychild] suddenly reappears above the tumor!</span>")
+				playsound(loc,'sound/effects/phasein.ogg', 200, 0, 50, TRUE, TRUE)
+				
+/obj/structure/elite_tumor/proc/fighters_check()
+	if(activity == TUMOR_ACTIVE && activator != null && activator.stat == DEAD || activity == TUMOR_ACTIVE && QDELETED(activator))
+		onEliteWon()
+	if(activity == TUMOR_ACTIVE && mychild != null && mychild.stat == DEAD || activity == TUMOR_ACTIVE && QDELETED(mychild))
+		onEliteLoss()
+	
+obj/structure/elite_tumor/proc/onEliteLoss()
+	playsound(loc,'sound/effects/tendril_destroyed.ogg', 200, 0, 50, TRUE, TRUE)
+	visible_message("<span class='boldwarning'>The tumor begins to convulse violently before beginning to dissipate.</span>")
+	mychild.myparent = null
+	if(activity == TUMOR_ACTIVE)
+		visible_message("<span class='boldwarning'>As the tumor closes, something is forced up from down below.</span>")
+		var/obj/structure/closet/crate/necropolis/tendril/lootbox = new /obj/structure/closet/crate/necropolis/tendril(loc)
+		if(boosted)
+			var/lootpick = rand(1, 2)
+			if(lootpick == 1 && mychild.loot_drop != null)
+				lootbox.contents += new mychild.loot_drop(lootbox.loc)
```

Comment:
I'm not sure what you tried to achieve because it's pretty mangled but if you just want item inside the lootbox
`new mychild.loot_drop(lootbox)`
same thing below